### PR TITLE
Create klouqz.json

### PR DESCRIPTION
**Pull request by klouqzz**

### CODE REVIEW

File: domains/klouqz.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Klouqz Website's",
+  "repo": "https://github.com/klouqzcreative/-klouqz.github.io",
+  "owner": {
+    "username": "Klouqz",
+    "email": "klouqz@gmail.com"
+  },
+  "record": {
+    "CNAME": "klouqz.github.io"
```
Comment (MaskDuck) [inline domains/klouqz.json:9]: 404?

File: domains/klouqz.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Klouqz Website's",
+  "repo": "https://github.com/klouqzcreative/-klouqz.github.io",
+  "owner": {
+    "username": "Klouqz",
```
Comment (MaskDuck) [inline domains/klouqz.json:5]: your github username?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/klouqz.json (hunk 2 above)
Comment (klouqzcreative) [reply domains/klouqz.json:5]: oh sorry

**Review by klouqzz — COMMENTED** [review]
{
  "description": "Klouqz Website's",
  "repo": "https://github.com/klouqzcreative/-klouqz.github.io",
  "owner": {
    "username": "klouqzcreative",
    "email": "klouqz@gmail.com"
  },
  "record": {
    "CNAME": "klouqz.github.io"
  }
}

**Pull request closed without merging**